### PR TITLE
Update README.md to point out which branches are in active usage.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+  
+  We use this branch to build the DEB package for the legacy repo.stackdriver.com repo.
+  
+* test-containers
+
+  We use this branch to store Dockerfiles that are used to build base containers from different distros with necessary dependencies installed so agents can be built in that environment.
+  
+* Development branches
+
+  All branches with GitHub username as prefixes are development branches.
```
Comment (igorpeshansky) [inline README.md]: Not only GitHub usernames, right? Maybe just `All branches with username prefixes are development branches.`?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
 # agent-packaging
 Combined packaging scripts for the Stackdriver monitoring agent
+
+## Branches in Use
+
+* master
```
Comment (igorpeshansky) [inline README.md]: [Optional] Use code font for branch names, e.g.:
```markdown
* `master`
  ...
* `stackdriver-agent-5.5.2`
  ...
```
to distinguish them from, e.g., "Development branches" below.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+  
+  We use this branch to build the RPM package for the legacy repo.stackdriver.com repo.
+  
+* legacy-agent-deb
+  
+  We use this branch to build the DEB package for the legacy repo.stackdriver.com repo.
+  
+* test-containers
+
+  We use this branch to store Dockerfiles that are used to build base containers from different distros with necessary dependencies installed so agents can be built in that environment.
```
Comment (igorpeshansky) [inline README.md]: [Optional] Split the long line in the markdown source...

Comment (igorpeshansky) [inline README.md]: [Optional] Make `Dockerfile` code font, e.g.: ``...store `Dockerfile`s that are...``.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,28 @@
...
+
+## Branches in Use
+
+* master
+
+  We use this branch to build the 6.X.X agent DEB and RPM packages for the Cloud Rapture repos.
+  
+* stackdriver-agent-5.5.2
+
+  We use this branch to build the 5.X.X agent DEB and RPM packages for the Cloud Rapture repos.
```
Comment (igorpeshansky) [inline README.md]: [Optional] `5.5.2-X agent DEB and RPM packages`...

File: README.md [hunk 5]
```diff
@@ -1,2 +1,28 @@
...
+
+* legacy-agent-rpm
+  
+  We use this branch to build the RPM package for the legacy repo.stackdriver.com repo.
```
Comment (igorpeshansky) [inline README.md]: [Optional] `the 5.5.2-X RPM package`...

Comment (igorpeshansky) [inline README.md]: [Optional] Make `repo.stackdriver.com` code font (here and below).

File: README.md [hunk 6]
```diff
@@ -1,2 +1,28 @@
...
+  
+* legacy-agent-deb
+  
+  We use this branch to build the DEB package for the legacy repo.stackdriver.com repo.
```
Comment (igorpeshansky) [inline README.md]: [Optional] `the 5.5.2-X DEB package`...

File: README.md (hunk 5 above)
Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (qingling128) [reply README.md]: Done

File: README.md [hunk 7]
```diff
@@ -1,2 +1,30 @@
...
+  We use this branch to build the RPM package for the legacy repo.stackdriver.com repo.
+
+* `legacy-agent-deb`
+
+  We use this branch to build the DEB package for the legacy repo.stackdriver.com repo.
+
+* `test-containers`
+
+  We use this branch to store `Dockerfile`s that are used to build base containers from different distros
+  with necessary dependencies installed so agents can be built in that environment.
```
Comment (igorpeshansky) [inline README.md]: Oops, just realized that this is actually describing the `build` directory. The `test-containers` branch will contain containers for reproducible testing of third-party application support.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,30 @@
 # agent-packaging
-Combined packaging scripts for the Stackdriver monitoring agent
+
+Combined packaging scripts for the Stackdriver monitoring agent.
+
+## Branches in Use
+
+* `master`
+
+  We use this branch to build the `6.X.X` agent DEB and RPM packages for the Cloud Rapture repos.
```
Comment (igorpeshansky) [inline README.md]: Another oops: s/Cloud Rapture/Google Cloud package repository/ everywhere.

**Comment by qingling128** [comment]
Updated. PTAL

File: README.md [hunk 9]
```diff
@@ -1,2 +1,30 @@
...
+  We use this branch to build the RPM package for the legacy repo.stackdriver.com repo.
+
+* `legacy-agent-deb`
+
+  We use this branch to build the DEB package for the legacy repo.stackdriver.com repo.
+
+* `test-containers`
+
+  We use the `build` directory in this branch to store `Dockerfile`s that are used to build containers for reproducible
+  testing of third-party application support.
```
Comment (igorpeshansky) [inline README.md]: How about just:
```
  This branch contains `Dockerfile`s for reproducible testing of third-party application support.
```
?

**Comment by qingling128** [comment]
Updated. PTAL

File: README.md (hunk 5 above)
Comment (igorpeshansky) [reply README.md]: Sorry, still not done, unless I'm missing a commit. I feel it's important to mention so we don't accidentally try to build a 6.0.0 package using those build scripts. WDYT?

File: README.md (hunk 6 above)
Comment (igorpeshansky) [reply README.md]: Sorry, still not done, unless I'm missing a commit. I feel it's important to mention so we don't accidentally try to build a 6.0.0 package using those build scripts. WDYT?

File: README.md (hunk 2 above)
Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (qingling128) [reply README.md]: Seems like in the last round I only replied `Done` for places I did not change... This one I'd prefer not to distinguish with finer granularity, as there is no plan to have another branch with `6.0.0` for legacy package. `legacy-agent-rpm` will be the only one for legacy RPM.

Comment (qingling128) [reply README.md]: Done now.

File: README.md (hunk 6 above)
Comment (qingling128) [reply README.md]: Seems like in the last round I only replied `Done` for places I did not change... This one I'd prefer not to distinguish with finer granularity, as there is no plan to have another branch with `6.0.0` for legacy package. `legacy-agent-deb` will be the only one for legacy DEB.

File: README.md (hunk 3 above)
Comment (qingling128) [reply README.md]: Done

Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (qingling128) [reply README.md]: Done

File: README.md [hunk 10]
```diff
@@ -1,2 +1,30 @@
 # agent-packaging
-Combined packaging scripts for the Stackdriver monitoring agent
+
+Combined packaging scripts for the Stackdriver monitoring agent.
+
+## Branches in Use
+
+* `master`
+
+  We use this branch to build the `6.X.X` agent DEB and RPM packages for the Google Cloud packaging repository repos.
```
[the remark this answers is not in the document]
Comment (qingling128) [reply README.md]: Done

File: README.md (hunk 9 above)
Comment (qingling128) [reply README.md]: Done

**Pull request merged**